### DISCUSSION TITLE
How to using php7.0

### DISCUSSION THREAD

**Discussion by sanker2008** [wardenenv/warden#682] [Help & Support]
I want to Install magento1 with php7.0
after modify .env to
...
NODE_VERSION=10
COMPOSER_VERSION=1
**PHP_VERSION=7.0**
PHP_XDEBUG_3=1
...

warden env up failed!

**Comment by bap14** [comment]
PHP 7.0 isn't built automatically by the Warden images process, so it's not available to use. You could download the build script(s) for warden images and build your own image, or find a third-party that has a pre-built PHP 7.0 or 5.5 image that has the extensions you need in it. Then just specify that image in the `.warden/warden-env.yml` file and Warden will use that image instead of the default image.

**Comment by navarr** [comment]
Go ahead and set `PHP_XDEBUG_3=0` - this will pull the (very old built) PHP 7 images.  These images do not support ARM64, and these versions of PHP are no longer supported.

If you want to run PHP 7.0 on ARM you will have to build custom images as @bap14 suggests.
